resources/validations/src/ssp-additions.sch

```diff
@@ -0,0 +1,1278 @@
+
+            <sch:p>inventory-item has only one scan-type property</sch:p>
+            <sch:assert
+                diagnostics="inventory-item-has-one-scan-type-diagnostic"
+                id="inventory-item-has-one-scan-type"
+                role="error"
+                test="count(oscal:prop[@name = 'scan-type']) = 1">inventory-item has only one scan-type property.</sch:assert>
+
+        </sch:rule>
+
+        <sch:rule
+            context="oscal:inventory-item[oscal:prop[@name = 'asset-type' and @value = ('os', 'infrastructure')]]">
+
+            <sch:p>"infrastructure" inventory-item has allows-authenticated-scan</sch:p>
+            <sch:assert
+                diagnostics="inventory-item-has-allows-authenticated-scan-diagnostic"
+                id="inventory-item-has-allows-authenticated-scan"
+                role="error"
+                test="oscal:prop[@name = 'allows-authenticated-scan']">"infrastructure" inventory-item has allows-authenticated-scan.</sch:assert>
+
+            <sch:p>"infrastructure" inventory-item has only one allows-authenticated-scan property</sch:p>
+            <sch:assert
+                diagnostics="inventory-item-has-one-allows-authenticated-scan-diagnostic"
+                id="inventory-item-has-one-allows-authenticated-scan"
+                role="error"
+                test="count(oscal:prop[@name = 'allows-authenticated-scan']) = 1">inventory-item has only one one-allows-authenticated-scan
+                property.</sch:assert>
+
+            <sch:p>"infrastructure" inventory-item has baseline-configuration-name</sch:p>
+            <sch:assert
+                diagnostics="inventory-item-has-baseline-configuration-name-diagnostic"
+                id="inventory-item-has-baseline-configuration-name"
+                role="error"
+                test="oscal:prop[@name = 'baseline-configuration-name']">"infrastructure" inventory-item has baseline-configuration-name.</sch:assert>
+
+            <sch:p>"infrastructure" inventory-item has only one baseline-configuration-name</sch:p>
+            <sch:assert
+                diagnostics="inventory-item-has-one-baseline-configuration-name-diagnostic"
+                id="inventory-item-has-one-baseline-configuration-name"
+                role="error"
+                test="count(oscal:prop[@name = 'baseline-configuration-name']) = 1">"infrastructure" inventory-item has only one
+                baseline-configuration-name.</sch:assert>
+
+            <sch:p>"infrastructure" inventory-item has a vendor-name property</sch:p>
+            <!-- FIXME: Documentation says vendor name is in FedRAMP @ns -->
+            <sch:assert
+                diagnostics="inventory-item-has-vendor-name-diagnostic"
+                id="inventory-item-has-vendor-name"
+                role="error"
+                test="oscal:prop[(: @ns = 'https://fedramp.gov/ns/oscal' and :)@name = 'vendor-name']">"infrastructure" inventory-item has a
```

Comment:
NB: Does this not mean `(: @ns = 'https://fedramp.gov/ns/oscal' and :)` `@ns` check is commented out. Is that intentional at this time?

Reply:
That is intentional and ephemeral.

There is discord between the FedRAMP-SSP-OSCAL-Template.xml and the related documentation. Sometimes just enumerated values; sometimes schema differences.

I can create an issue regarding multiple instances of discord if that is an appropriate manner in which to resolve such.

Reply:
Ok, well yes then. It is really a question more about upstream data (in GSA/fedramp-automation and usnistgov/OSCAL). I guess we can resolve it as necessary, but some things moved upstream and that will make the situation for us more difficult in certain contexts, but not this one.